Tasks/app-store-release/app-store-release.ts

```diff
@@ -134,15 +134,25 @@ async function run() {
         tl.debug('Checking for ruby install...');
         tl.which('ruby', true);
 
-        //Whenever a specific version of fastlane is requested, we're going to uninstall all installed
-        //versions of fastlane beforehand.  Note that this doesn't uninstall dependencies of fastlane.
-        if (installFastlane && fastlaneVersionToInstall) {
+        // check for previous fastlane first
+        function previousVersionCheck () {
+            tl.debug('Checking for previous fastlane version first...')
             let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
-            gemRunner.arg(['uninstall', 'fastlane']);
-            tl.debug(`Uninstalling all fastlane versions...`);
-            gemRunner.arg(['-a', '-I']);  //uninstall all versions
-            await gemRunner.exec();
+            gemRunner.arg(['list', 'fastlane'])
+            gemRunner.arg(['-i']);
```

Comment:
This doesn't look like it will work. gemRunner.exec() has to be run and the output needs to be checked for true or false.

Reply:
Using the -i flag the output produced is either true or false, but I did miss gemRunner.exec(). I updated now.

Reply:
The build failed (linter issues) but I am pretty sure this will not work. exec returns the return code of the command which will is likely to be 0, so this will never be true. I know how to fix this, the output of the command needs to be parsed. I will update today.